src/main/java/me/zeroeightsix/kami/module/modules/combat/Aura.java

```diff
@@ -178,4 +190,8 @@ private boolean canEntityFeetBeSeen(Entity entityIn) {
         return mc.world.rayTraceBlocks(new Vec3d(mc.player.posX, mc.player.posY + mc.player.getEyeHeight(), mc.player.posZ), new Vec3d(entityIn.posX, entityIn.posY, entityIn.posZ), false, true, false) == null;
     }
 
+    private enum WaitMode {
+        KAMI, TICK
```

Comment:
`KAMI` doesn't really explain what this mode does: consider `Delay`/`Wait`/... ?

Reply:
How do you like `Dynamic` and `Static` for wait modes?

Reply:
changed [here](https://github.com/blockparole/KAMI/commit/fcdde15967ef2b544caa4f159415cb0b0c2b928a)